### ISSUE TITLE
Noen tekster skjult for bruker som blir lest opp av skjermleser blir ikke oversatt til engelsk av språkvelgeren. 

### ISSUE THREAD

**Issue by EirikOlavHaugenSSB** [Altinn/app-frontend-react#2279]
### Description of the bug

I en tjeneste med norsk og engelsk språk og språkvelger satt på. Vi bruker skjermleseren NVDA til å lese opp skjema for oss. Tekstene er oversatt i språkfila og bytter riktig når vi velger engelsk i språkvelgeren. Tekster som sidetall blir lest riktig opp.
Derimot blir ikke tekster som "skrivebeskyttet felt" og "overskriftsnummer" oversatt til engelsk og skjermleseren leser opp norske tekster med engelsk tolkning.

### Steps To Reproduce

1. Lag skjema med engelsk og norsk side (kan hende engelsk holder)
2. Lag en input-komponent som er skrivebeskyttet, og en overskriftskomponent. 
3. Bruk språkvelger til å velge engelsk
4. Bruk NVDA til å få tekster lest opp og naviger til input-feltet og overskriftkomponenten
5. Tekster blir lest opp på norsk istedenfor engelsk selv om engelsk språk er valgt. 

### Additional Information

Eksempelrepo. Første side er på engelsk og bokmål.
Repo: https://altinn.studio/repos/EirikOlavHaugenSSB/ra0759-01
Commit ID: d6d1786dd1ab460da77c4b58c467328e01c5a298

Diskusjon:
https://altinn.slack.com/archives/C02EJ9HKQA3/p1723102447977729

**Comment by EirikOlavHaugenSSB** [comment]
Det er andre tekster enn nevnt her som også ikke er oversatt riktig. 
Fant: "radioknapp", "avkrysningsboks", "krysset", "avkrysset", "knapp"

**Comment by olavflar** [comment]
Vi skal teste denne med en Mac/pc som har systemspråk på engelsk og se videre på dette

**Comment by olavflar** [comment]
Når det er løst så ligger det en guide her for å oppdatere tilgjengelighetserklæringen: https://digdir.atlassian.net/wiki/x/CgDy-/
